package.json

```diff
@@ -153,13 +153,19 @@
     "lint": "eslint --cache .",
     "lint:fix": "eslint --cache --fix ."
   },
+  "dependencies": {
```

Comment:
Please, don't add dependencies to devDependencies, they won't be bundled otherwise

Reply:
Wait, I never added them to dev-dependencies. They are added as real dependencies.

Reply:
I wanted to write in the first comment "don't add to "dependencies", use "devDependencies""

Reply:
Got it. I applied it now.